### ISSUE TITLE
Bug: `--enclosing-braces` makes values invalid if they contain escaped characters

### ISSUE THREAD

**Issue by chrisgrieser** [FlamingTempura/bibtex-tidy#423]
```bash
cat test.bib | bibtex-tidy --enclosing-braces=title
```

Using `--enclosing-braces` results in escaped characters, such as `{\"a}`, having their leading `{` removed, making the whole bibtex entry invalid.

```bib
@incollection{BlattelMink2021,
	title = {{Das Innovationsverst{\"a}ndnis von Joseph A. Schumpeter}},
	author = {Bl{\"a}ttel-Mink, Birgit},
}

% becomes
@incollection{BlattelMink2021,
  title         = {{Das Innovationsverst\"a}ndnis von Joseph A. Schumpeter}},
  author        = {Bl{\"a}ttel-Mink, Birgit}
}
```

**Comment by jdujava** [comment]
I can confirm this behavior. Also `--remove-braces` leads to a similar invalid entry.

**Comment by FlamingTempura** [comment]
Fixed in v1.14.0
